### PR TITLE
New package: Microsoft.Azure.QuickReview version 0.16.0

### PR DESCRIPTION
**Pull request by mloskot**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107907)

image

### CODE REVIEW

File: manifests/m/Microsoft/Azure/QuickReview/0.16.0/Microsoft.Azure.QuickReview.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+PublisherSupportUrl: https://github.com/Azure/azqr/issues
+# PrivacyUrl:
+# Author:
+PackageName: Microsoft Azure Quick Review
+# PackageUrl:
+License: MIT
+LicenseUrl: https://github.com/Azure/azqr/blob/v.0.16.0/LICENSE
+Copyright: Copyright (c) Microsoft Corporation
+# CopyrightUrl:
+ShortDescription: A tool to check complaince of Azure resources with Azure best practices and recommendations.
```
Comment (Trenly) [inline manifests/m/Microsoft/Azure/QuickReview/0.16.0/Microsoft.Azure.QuickReview.locale.en-US.yaml:18]:
```suggestion
ShortDescription: A tool to check compliance of Azure resources with Azure best practices and recommendations.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/Azure/QuickReview/0.16.0/Microsoft.Azure.QuickReview.locale.en-US.yaml (hunk 1 above)
Comment (mloskot) [reply manifests/m/Microsoft/Azure/QuickReview/0.16.0/Microsoft.Azure.QuickReview.locale.en-US.yaml]: Thx, fixed

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
Usage:
  azqr [flags]
  azqr [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  rules       Print all azqr rules
  scan        Scan Azure Resources

Flags:
...
```

**Review by stephengillie — APPROVED** [review]

**Comment by mloskot** [comment]
@stephengillie
> This package is expecting arguments: ...

Does such package need any special handling, flagging in the manifests that I perhaps have missed?

**Comment by stephengillie** [comment]
> @stephengillie TBH, this seems to me like a kind of a hack. I would have never figured that for a CLI tool that offers interactive mode, I should utilise this
> 
> [Commands: # Optional commands or aliases to run the package](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md)
> 
> in order to ease running it non-interactively (perhaps by winget's own pipelines).
> 
> It would be better if the manifest included something like `VersionCommand` where packagers could stick "full command to obtain package version to stdout" e.g `VersionCommand: gron --version` then it could be referenced by anything non-interactive that need such command to ensure runnability of packaged software (e.g. pipelines).

This is why I merely approved and noted - fixing becomes out of scope, and should be addressed through an issue or other means.

**Comment by mloskot** [comment]
or "better is an enemy or good", yes I get that and I don't disagree. I just aimed to get the context and confirm my own understanding of why that suggestion was made.

**Pull request merged**